docker-extension/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  headlamp:
+    image: ghcr.io/kinvolk/headlamp:v0.12.0
```

Comment:
I am confused if the GitHub action to build and push the extension should get auto-triggered on every release or be manually triggered. Since the image version has to be updated every time before the release and there is a high chance that this gets missed out.

Reply:
I think we're aiming to /automate all the things/ eventually, so probably good to set it up for on release.

Reply:
In that case, we will have to make the updating the tag and create a commit also as a step in GitHub action.

Reply:
let me check how feasible it is.

Reply:
This part could be done later? Joaquim has a better overview of the release process... and also where it is going.

Reply:
Added steps to update the tag automatically and commit.
sharing a test run from my fork for reference:
https://github.com/yolossn/headlamp/runs/8224275472?check_suite_focus=true

Reply:
> This part could be done later? Joaquim has a better overview of the release process... and also where it is going.

Have added the steps as discussed, I think we can wait till Joaquim is back to get feedback.